cmd/clotas/commands/run.go

```diff
@@ -0,0 +1,61 @@
+// Package commands /*
+package commands
+
+/*
+Copyright © 2021 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import (
+	"github.com/davidlukac/go-clotas/internal/pkg/clotas"
+	"github.com/spf13/cobra"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+// runCmd represents the run command
+var runCmd = &cobra.Command{
+	Use:   "run",
+	Short: "Run Clota scripts",
+	Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command. For example:
+
+Cobra is a CLI library for Go that empowers applications.
+This application is a tool to generate the needed files
+to quickly create a Cobra application.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		files := clotas.GetFiles(clotas.DefaultTargetFolder)
+		for _, f := range files {
+			cmd := exec.Command(filepath.Join(clotas.DefaultTargetFolder, f))
```

Comment:
I would add some comment here, it's a bit confusing without deeper knowledge of the code